setup.cfg

```diff
@@ -18,7 +18,7 @@ classifiers =
     License :: OSI Approved :: MIT License
     Natural Language :: English
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.7
```

Comment:
You should add one line per python version:

```suggestion
    Programming Language :: Python :: 3.7
    Programming Language :: Python :: 3.8
    Programming Language :: Python :: 3.9
    Programming Language :: Python :: 3.10
    Programming Language :: Python :: 3.11
```

Reply:
Whoops you're right!